Vehicle/Vehicle/doc/spec.md

```diff
@@ -0,0 +1,169 @@
+# Vehicle
+
+## Description
+
+A vehicle.
+
+## Data Model
+
++ `id` : Entity's unique identifier. 
+
++ `type` : Entity type. It must be equal to `Vehicle`.
+
++ `name` : Name given to this vehicle. 
+    + Normative References: [https://schema.org/name](https://schema.org/name)
+    + Optional
+
++ `description` : Vehicle description. 
+    + Normative References: [https://schema.org/description](https://schema.org/description)
+    + Optional
+
++ `location` : Vehicle's last known location represented by a GeoJSON Point. 
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Attribute metadata:
+        + `timestamp`: Timestamp which captures when the vehicle was at that location.
+        This value can also appear as a FIWARE [TimeInstant](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
+            + Type: [DateTime](http://schema.org/DateTime) or `ISO8601` (legacy).   
+    + Mandatory
```

Comment:
well, we can leave it as optional, certainly, although I don't know if a vehicle not located makes sense at all :)